src/MockClassifier.UnitTests/InstitutionControllerTests.cs

```diff
@@ -12,10 +13,14 @@ namespace MockClassifier.UnitTests
     public class InstitutionControllerTests
     {
         private readonly InstitutionController sut;
+        private readonly Mock<IDmrService> dmrService = new();
```

Comment:
Quick question, do we need to create object here if it is being assigned in line 22? please ignore if comment is redundant thought because I probably need to go over my .Net testing framework notes again 😆